test/index-test.js

```diff
@@ -110,6 +110,14 @@ tests = [{
     m: 'i'
   , o: [{i: 1, o: 2}, {i: 2, o: 2}]
   , e: [{i: 1}, {i: 2}]
+}, {
+    m: 'foo(bar)'
+  , o: { foo: { biz: 'baz' } }
+  , e: { foo: {} },
```

Comment:
We also use this library for object masking. The motivation was exactly the use case you mentioned -- we have no way to distinguish an input value of `null` or `{}`. For that reason we prefer that `json-mask` respects the input value type.